src/messaging/ExchangeMgr.cpp

```diff
@@ -99,35 +99,33 @@ CHIP_ERROR ExchangeManager::Shutdown()
     return CHIP_NO_ERROR;
 }
 
-ExchangeContext * ExchangeManager::NewContext(const NodeId & peerNodeId, void * appState)
+ExchangeContext * ExchangeManager::NewContext(const NodeId & peerNodeId, ExchangeContextDelegate * delegate)
```

Comment:
Will this delegate same as SetDelegate(ExchangeContextDelegate * delegate)? if so, how to guarantee the protocol not make the mistake if we ask the protocol to set the same delegate in different contexts.

Reply:
Please remove SetDelegate API to prevent protocol from setting different delegate